.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Test pyblock
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip coverage nose numpy matplotlib pandas
+          python -m pip install -e .
+      - name: Test with nose
+        run: |
+          python -m nosetests --with-coverage --cover-package=pyblock --cover-min-percentage=85 pyblock
```

Comment:
Maybe switch to `pytest`  instead of nose here? What do you think?

https://stackoverflow.com/questions/59420123/is-there-a-standard-way-to-fail-pytest-if-test-coverage-falls-under-x suggests we have to run the tests and then separately test the coverage is over x%, but that's ok.

Reply:
I am happy to do the `pytest` switch, but I'd prefer it going into a separate PR. I'd like to first make sure that the GHA workflows runs as expected.

On the topic of coverage: with `pytest` I usually run it at once and then send the data to Codecov for reporting and visualization.

Reply:
Fair.